src/plugins/event_annotation_listing/public/components/group_editor_flyout/group_preview.test.tsx

```diff
@@ -20,14 +20,14 @@ import {
   EmbeddableComponent,
```

Comment:
Something is being malformed when it comes to saved object state for by value objects. On this PR, I created this dashboard:
<img width="1295" alt="Screenshot 2024-11-06 at 12 44 35" src="https://github.com/user-attachments/assets/b7cc6134-2c2e-4b2e-b483-b5cecdc0a696">
[export.json](https://github.com/user-attachments/files/17646565/export.json)

and then I ran this saved object in main (or switch branches), I get a bunch of errors in the console:
<img width="1293" alt="Screenshot 2024-11-06 at 12 44 12" src="https://github.com/user-attachments/assets/5a95a7cf-cff2-4725-afc0-2356fbec7727">

Is this intentional? Should anything from saved object persisted state change?

Reply:
I'm currently investigating this bug.

Reply:
Fixed via [3b982d9](https://github.com/elastic/kibana/pull/186642/commits/3b982d9bb5735b177e0f9e807596ee90195934dc) and [ce07080](https://github.com/elastic/kibana/pull/186642/commits/ce070808f7cfc04ab3494bfacb7b2de08f5d672c). Added tests at unit level with a8f49af

Reply:
Thanx Marco, I havem't tested it but I can tomorrow.